pkgs/development/compilers/elm/packages/node/generate-node-packages.sh

```diff
@@ -10,3 +10,5 @@ $(nix-build $ROOT -A nodePackages.node2nix --no-out-link)/bin/node2nix \
     -o node-packages.nix \
     -c node-composition.nix \
     --no-copy-node-env -e ../../../../node-packages/node-env.nix
+# well, elm-pages requires two different version of esbuild so we twist it's wrist to only use one
+sed -i 's/sources."esbuild-0.19.12"/sources."esbuild-0.21.5"/' node-packages.nix
```

Comment:
same here

Reply:
Not sure how to do this, since this is a shell script and it doesn't have `substituteInPlace`.